dashboard/config/locales/data.en.yml

```diff
@@ -509,10 +509,13 @@ en:
         'hoc2015_preview_blocks' : 'Make your own game blocks'
         'hoc2015_preview_typing' : 'Make your own game typing'
         'hoc2015_preview_conclusion' : 'Make your own game conclusion'
-        'starwars_intro' : 'Introduction'
-        'starwars_typing' : 'JavaScript'
-        'starwars_events' : 'Events'
-        'starwars_congrats' : 'Congratulations'
+        'starwars_intro' : 'Star Wars Hour of Code - Introduction'
```

Comment:
The official name is Star Wars: Building a Galaxy With Code.. for what it's worth.